test/functionalTest/cases/0000_json_parse/jsonParseIndividualContextEntity.test

```diff
@@ -165,17 +165,20 @@ Date: REGEX(.*)
 }
 4: +++++++++ PUT that fails +++++++++++++++++++
 HTTP/1.1 200 OK
-Content-Length: 172
+Content-Length: 72
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "errorCode": {
-        "code": "472",
-        "details": "action: UPDATE - entity: [ENTITY_ID, ] - offending attribute: temperature",
-        "reasonPhrase": "request parameter is invalid/not allowed"
-    }
+    "contextResponses": [
+        {
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
```

Comment:
It seems that the modifications in the PR have introduced a change in the behaviour of NGSIv1 resquest. Some 472 errors are now 200 OK responses.

NGSIv1 is a deprecated API, which relaxes the requirement for backward compatibility. However, I'd like to understand why this is happening before taking the decision of changing the behaviour. If we can keep the behaviour with small changes, it would be better doing that way.

Reply:
> It seems that the modifications in the PR have introduced a change in the behaviour of NGSIv1 resquest. Some 472 errors are now 200 OK responses.
> 
> NGSIv1 is a deprecated API, which relaxes the requirement for backward compatibility. However, I'd like to understand why this is happening before taking the decision of changing the behaviour. If we can keep the behaviour with small changes, it would be better doing that way.

I have tried to fix this issue in another way by using the existing parameters attributeAlreadyExistsError and attributeAlreadyExistsList instead of adding new. But it also introduced change in NGSIv1 resquest. I have not found any other way to fix this issue. Do you have any idea how we can fix this without impacting the backward compatibility?

Reply:
I'll test with your code and try to find why this is happening because, just looking the PR, I'd say that everything is correct.

Reply:
It seems that you modifications in commit e97d1fdd8bc194407df722a2783bf8a411514c1b fixed the test that were failing due to NGSIv1 response changes. Great!

However, it seems that some test are still failing after that. Do you have any clue of them? Any common failing pattern in all them?

Reply:
> However, it seems that some test are still failing after that. Do you have any clue of them? Any common failing pattern in all them?

I have fixed some test cases in my latest commit  https://github.com/telefonicaid/fiware-orion/pull/3900/commits/355d49316c8e79c58a656ac506c6ead7e0a40f07 As per my analysis these test case need fix as per modification done in PR

I have observed some common failing pattern like some error of `"PartialUpdate"` is now giving `"Unprocessable"` response. Some test cases the error of `"No context element found"` is now showing `"Internal Server Error"` and some test case having ContexProvider (CP) are failing. I have not found the cause of these failing test cases.

Reply:
Try removing line 3113 (`eP->attributeVector[ix]->skip = true;`). If skip is set to true, the registration lookup for not found attributes is disabled, so many of the forwarding test cases are going to fail (although it doesn't fix all the current failing cases...).

In addition, it could be great if you could update this PR branch with master.

Reply:
> Try removing line 3113 (`eP->attributeVector[ix]->skip = true;`). If skip is set to true, the registration lookup for not found attributes is disabled, so many of the forwarding test cases are going to fail (although it doesn't fix all the current failing cases...).
> 
> In addition, it could be great if you could update this PR branch with master.

Fixed in https://github.com/telefonicaid/fiware-orion/pull/3900/commits/04eb1706ecd3dc5af2512a33461dbda2f8ffdfe0

Some test cases still failing:
```
   cases/3068_cprs_full_functional/fwd_v2_update_forward_depends_on_attr_field.test
   cases/3068_cprs_full_functional/fwd_v2_update_found_and_not_found_in_local_and_three_cprs.test
   cases/3068_cprs_full_functional/fwd_v2_update_three_found_on_cps_two_local_four_not_found.test
   cases/3068_cprs_full_functional_v2/fwd_v2_ngsiv2_update_forward_depends_on_attr_field.test
   cases/3068_cprs_full_functional_v2/fwd_v2_ngsiv2_update_found_and_not_found_in_local_and_three_cprs.test
   cases/3068_cprs_full_functional_v2/fwd_v2_ngsiv2_update_three_found_on_cps_two_local_four_not_found.test
   cases/1127_v1_append_strict/v1_append_strict.test
```

Reply:
I think the following modification

```diff
--- a/src/lib/serviceRoutines/postUpdateContext.cpp
+++ b/src/lib/serviceRoutines/postUpdateContext.cpp
@@ -339,7 +339,8 @@ static bool updateForward
       upcrsP->fill(upcrP, SccOk);
       return true;
     }
-    if (statusCode == SccContextElementNotFound)
+    // SccInvalidModification is the case por partial updates
+    if ((statusCode == SccContextElementNotFound) || (statusCode == SccInvalidModification))
     {
       upcrsP->fill(upcrP, SccContextElementNotFound);
       return true;
```

would solve the following test cases:

```
cases/3068_cprs_full_functional/fwd_v2_update_forward_depends_on_attr_field.test
cases/3068_cprs_full_functional/fwd_v2_update_found_and_not_found_in_local_and_three_cprs.test
cases/3068_cprs_full_functional/fwd_v2_update_three_found_on_cps_two_local_four_not_found.test
cases/3068_cprs_full_functional_v2/fwd_v2_ngsiv2_update_forward_depends_on_attr_field.test
cases/3068_cprs_full_functional_v2/fwd_v2_ngsiv2_update_found_and_not_found_in_local_and_three_cprs.test
cases/3068_cprs_full_functional_v2/fwd_v2_ngsiv2_update_three_found_on_cps_two_local_four_not_found.test
```

Reply:
Thank you for your help and guidance I have modify code as suggested by you in https://github.com/telefonicaid/fiware-orion/pull/3900/commits/7c81f6968698b89cede917e40266ed29105fdc1f